website/static/js/registerNode.js

```diff
@@ -5,29 +5,46 @@
 var $ = require('jquery');
 var bootbox = require('bootbox');
 
-var preRegisterMessage =  function(title, parentTitle, parentUrl) {
-    return 'You are about to Register the component "' + title +
-        '" and everything that is inside it. This will not register' +
-        ' your larger project "' + parentTitle + '" and its other components.' +
-        ' If you want to register the entire project, please go <a href="' +
-        parentUrl + '">here.</a>';
+
+var preRegisterMessage =  function(title, parentTitle, parentUrl, category) {
+    if (parentUrl) {
+        return 'You are about to register the ' + category + ' <b>' + title +
```

Comment:
A few minor changes to make, which I'll make when I merge:

- Make **not** bold
- Add comma: "This will not register your larger project, ..."
- "This will not register your larger project.." --This could be inaccurate (i.e. if the parent is a component). Not exactly sure what the best fix is. We might want to say "This will not register any parents" to be as general as possible. Or maybe: "This will not register the parent, <Project Name>". If you would ..." ping @lbanner @bnosek for feedback